<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_appname{
font-size: 28px;
font-family: &quot;思源黑体 CN Medium&quot;;
font-weight: 400;
color: #FFFFFF;
}
#closeButton
{
font-size: 18px;
font-family: &quot;思源黑体 CN Medium&quot;;
font-weight: 500;
}
#lineEdit_loginName,#lineEdit_password{
background: rgba(5,91,201,0.08);
border: 2px solid rgba(5,91,201,0.2);
font-size: 24px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: 400;
color: #FFFFFF;
selection-background-color: #1167CF;
padding-left:0px;
}
#lineEdit_loginName:hover,#lineEdit_password:hover{
background: rgba(5,91,201,0.2);
border: 2px solid rgba(5,91,201,0.2);
}
#lineEdit_loginName:focus,#lineEdit_password:focus{
background: rgba(5,91,201,0.08);
border: 2px solid #055BC9;
}
#widget_container{
background-repeat:no-repeat;
background-position:center;
}
#label_container_name{
font-size: 32px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color: #055BC9;
}
#verticalWidget,#widget,#widget_3,#widget_5,#label,#label_LoginWelcomeChinese{
background:transparent;
}
.Login{

background-repeat:no-repeat;
background-position:center;
margin-left:-2px;
margin-bottom:-4px;
}
#label_logo{

background-repeat:no-repeat;
background-position:center;
}
#label_LoginWelcomeChinese{
font-size: 66px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color: #FFFFFF;
}
#label_LoginWelcomeEnglish{
font-size: 26px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: 400;
color: #737989;
}
#label{
font-size: 18px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: 400;
color: #6A7384;
}
#pushButton_login{
font-size: 26px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color:white;
margin-left:-2px;
}
#pushButton_login:hover{
font-size: 26px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color:white;
}
#pushButton_login:pressed{
font-size: 26px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color:white;
}
#pushButton_login:disabled{
font-size: 26px;
font-family:&quot;思源黑体 CN Medium&quot;;
font-weight: bold;
color:#DBE3F7;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>79</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>266</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>116</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>116</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>27</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_appname">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_logo">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/Resources/Logo_nt1.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>34</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>关机</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="class" stdset="0">
         <string>BlackButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>271</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="class" stdset="0">
         <string>login</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_LoginWelcomeChinese">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>口腔根管手术导航定位系统</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="class" stdset="0">
            <string>login</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>48</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_container" native="true">
           <property name="minimumSize">
            <size>
             <width>631</width>
             <height>437</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>631</width>
             <height>437</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>58</number>
            </property>
            <property name="topMargin">
             <number>35</number>
            </property>
            <property name="rightMargin">
             <number>58</number>
            </property>
            <property name="bottomMargin">
             <number>65</number>
            </property>
            <item>
             <widget class="QLabel" name="label_container_name">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>用户登录</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>35</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_loginName">
              <property name="minimumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>请输入登录ID</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>26</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_password">
              <property name="minimumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;禁止输入 /:*?&amp;quot;&amp;lt;|'%&amp;gt;&amp;amp; 等特殊字符&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>请输入密码</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>16</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="checkBox_rememberMe">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>记住密码</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>35</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_login">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>515</width>
                <height>56</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>思源黑体 CN Medium</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>登录</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_loginName</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../qr.qrc"/>
 </resources>
 <connections/>
</ui>
